fairlearn/metrics_plots/plot_disparities_in_performance.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from sklearn.metrics import accuracy_score
+from fairlearn.metrics import group_summary, false_positive_rate, false_negative_rate
+
+
+def plot_disparities_in_performance(y_true, y_pred, sensitive_features):
+    """Plot disparities in performance for a binary classifier with a single binary sensitive feature.
+
+    This helps visualize differences in overall prediction error, as well as false positive
+    and false negative error rates across the binary values of a sensitive attribute.
+
+    :param 1D-array y_true: Ground truth (correct) labels.
+    :param 1D-array y_pred: Predicted labels :math:`h(X)` returned by the classifier.
+    :param 1D-array sensitive_features: Sensitive features.
+    :return: None (shows a matplotlib plot)
+    """
+    # compute
+    accuracy_summary = group_summary(
+        accuracy_score,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    fp_summary = group_summary(
+        false_positive_rate,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    fn_summary = group_summary(
+        false_negative_rate,
+        y_true,
+        y_pred,
+        sensitive_features=sensitive_features)
+    sensitive_values = sensitive_features.unique()
+    overpredictions = []
+    underpredictions = []
+    for sensitive_value in sensitive_values:
+        overpredictions.append(fp_summary['by_group'][sensitive_value])
+        underpredictions.append(fn_summary['by_group'][sensitive_value])
```

Comment:
See the issue description for more, but a review of the code in this area that determines  overpredictions and underpredictions would be super helpful! 👍

Reply:
This feels like something that might belong in the metrics code (although that is in the process of being rewritten).

Reply:
I pointed this out in the other comment (today) but after consulting with @MiroDudik overprediction != FPR and underprediction != FNR, so we should change it so that it doesn't say over and underprediction anymore, but rather FPR and FNR.

Reply:
@romanlutz Thanks!  This is super helpful 👍 

As the [current plan for 0.5](https://hackmd.io/1b-RSHfQR221tokjL3j3JA?edit) is to ship the metrics changes, switch to Flask for the widget, and make documentation updates all around (eg, so the quickstart works), I'll leave this work alone so as to not slow down those other things from shipping first.